app/scripts/modules/core/authentication/userMenu/userMenu.directive.html

```diff
@@ -0,0 +1,3 @@
+<li ng-show="authEnabled" class="user-menu">
```

Comment:
You could wrap the `<user-menu></user-menu>` tag in another tag, e.g. `<div><user-menu></user-menu></div>`, then test if the parent node contains an `<li>`. Give that a shot and see if it works okay.